cmd/argo/commands/common/get_test.go

```diff
@@ -15,12 +16,32 @@ import (
 	"github.com/argoproj/argo-workflows/v3/workflow/util"
 )
 
+var (
+	workflowName string = "testWF"
+)
+
+func init() {
+	// these values get used as part of determining node name and would normally be set as part of
+	// running the application
+	JobStatusIconMap = map[wfv1.NodePhase]string{
+		wfv1.NodePending:   ansiFormat("Pending", FgYellow),
```

Comment:
Use `wfv1.NodePending` instead of writing out the actual string?

Reply:
Sorry, I'm not sure I'm following. What's the recommendation? As you probably know, I'm setting this so it can be used [here](https://github.com/argoproj/argo-workflows/blob/master/cmd/argo/commands/common/get.go#L468).